[2022-01-25 11:07:34,299] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.load_data 2022-01-25T11:02:24.583418+00:00 [queued]>
[2022-01-25 11:07:34,324] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.load_data 2022-01-25T11:02:24.583418+00:00 [queued]>
[2022-01-25 11:07:34,324] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 11:07:34,325] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-01-25 11:07:34,326] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 11:07:34,348] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): load_data> on 2022-01-25T11:02:24.583418+00:00
[2022-01-25 11:07:34,355] {standard_task_runner.py:52} INFO - Started process 16830 to run task
[2022-01-25 11:07:34,362] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'classification', 'load_data', '2022-01-25T11:02:24.583418+00:00', '--job-id', '114', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/classification.py', '--cfg-path', '/tmp/tmpt1i7dmdm', '--error-file', '/tmp/tmpf33ijlpi']
[2022-01-25 11:07:34,370] {standard_task_runner.py:77} INFO - Job 114: Subtask load_data
[2022-01-25 11:07:34,436] {logging_mixin.py:104} INFO - Running <TaskInstance: classification.load_data 2022-01-25T11:02:24.583418+00:00 [running]> on host 54a54d544bf0
[2022-01-25 11:07:34,490] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=classification
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2022-01-25T11:02:24.583418+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-25T11:02:24.583418+00:00
[2022-01-25 11:07:55,618] {python.py:118} INFO - Done. Returned value was: None
[2022-01-25 11:07:55,651] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=classification, task_id=load_data, execution_date=20220125T110224, start_date=20220125T110734, end_date=20220125T110755
[2022-01-25 11:07:55,758] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-01-25 11:07:55,793] {local_task_job.py:146} INFO - Task exited with return code 0
